### ISSUE TITLE
Set `spellcheck` property in Electron `webPreferences` instead of attribute

### ISSUE THREAD

**Issue by alexwkleung** [alexwkleung/Iris#54]
Electron natively can disable or enable the `spellcheck` property so you don't need to set the attribute of the element node.

Example inside `index.ts`

```ts 
let mainWindow: BrowserWindow = {} as BrowserWindow;

  //check if platform is darwin
  if(process.platform === 'darwin') {
      //log
      console.log("Platform is darwin (macOS)");

      mainWindow = new BrowserWindow({
          width: 1200,
          height: 750,
          show: false,
          autoHideMenuBar: true,
          titleBarStyle: 'hiddenInset',
          webPreferences: {
          preload: join(_dirname, '../preload/index.js'),
          sandbox: false,
          spellcheck: false //set spellcheck property
        }
    });
  }
```

**Comment by alexwkleung** [comment]
See f9f5824 for the commit.

**Comment by alexwkleung** [comment]
Fixed commit message in 62f35af
